pkgs/applications/science/math/R/default.nix

```diff
@@ -61,6 +61,12 @@ stdenv.mkDerivation rec {
     echo >>etc/Renviron.in "TZDIR=${tzdata}/share/zoneinfo"
   '';
 
+  postBuild = stdenv.lib.optionalString stdenv.isDarwin ''
+    sed -i etc/Makeconf \
```

Comment:
Does R not have some `configure` flag that allows us to configure those paths? Messing with the internals of the build state feels somewhat fragile.

Reply:
Yes R does. `./configure --help` says
``` 
CPPFLAGS    (Objective) C/C++ preprocessor flags, e.g. -I<include dir> if
              you have headers in a nonstandard directory <include dir>
LDFLAGS     linker flags, e.g. -L<lib dir> if you have libraries in a
              nonstandard directory <lib dir>
```